tests/scripts/install.sh

```diff
@@ -26,7 +26,7 @@ if [ -z "$PULL_NUMBER" ]; then
 else
   curl -O -L https://github.com/${REPO_OWNER}/${REPO_NAME}/pull/${PULL_NUMBER}.patch
   echo "Applying followng patch:"
-  cat ${PULL_NUMBER}.patch
+  cat ${PULL_NUMBER}.patch | tee ${ARTIFACT_DIR}/github-pr-${PULL_NUMBER}.patch
```

Comment:
Should we rather `cp` it to the `ARTIFACT_DIR` than `tee`?

Reply:
Is there any need to print the patch in the main test log if we are saving it as an artifact?

Reply:
After a chat with @crobby, I have removing the output of logs/files/... that we are saving as artifacts